src/pdf/DriverOverview/DriverOverviewPdfButton.tsx

```diff
@@ -43,7 +46,32 @@ type ParcelsForDeliveryResponse =
           error: { type: ParcelsForDeliveryErrorType; logId: string };
       };
 
-type ParcelsForDeliveryErrorType = "parcelFetchFailed" | "noMatchingClient";
+type ParcelsForDeliveryErrorType = "parcelFetchFailed" | "noMatchingClient" | "eventFetchFailed";
+
+type ParcelEventResponse =
+    | {
+          data: Schema["events"][];
+          error: null;
+      }
+    | {
+          data: null;
+          error: { type: ParcelsForDeliveryErrorType; logId: string };
+      };
+
+// Gets all "shipping labels downloaded" events for the given parcels
+const getParcelEvents = async (parcelIds: string[]): Promise<ParcelEventResponse> => {
```

Comment:
if this is used for multiple parcels, we should name it getParcelsEvents or something to indicate that (as well as changing it to be more descriptive anyway)